### DISCUSSION TITLE
transparency of header and footer images

### DISCUSSION THREAD

**Discussion by bhattacharya-a-bt** [greenelab/lab-website-template#206] [Q&A] [answered]
I'm hoping this is a question that can be answered very quickly. I'm trying to adjust the transparency of the header and footer images, but I can't seem to find the right parameter to adjust this. Any help would be greatly appreciated.

**Comment by vincerubinetti** [comment]
Yes this one's a little tricky to find. You want to change the `opacity` value in the `.background:before` style in `_styles/background.scss`.

**Reply by vincerubinetti** [reply, marked as answer]
Just make sure your text on top of the header/footer has enough contrast with whatever background image you're using. The default styles in the template are set the way they are to try to ensure that this is always the case, no matter what type of image the user provides. But as a result, you may find the images being too dim, depending on their specifics.
